envi/archs/arm/__init__.py

```diff
@@ -9,8 +9,15 @@
 from envi.archs.arm.disasm import *
 
 class ArmModule(envi.ArchitectureModule):
+    '''
```

Comment:
please don't use this form of comment for this.  instead, use # to start each line.
the use of '''  ''' immediately following a class or function line is known as a "docstring" and is part of python's self-documention.